### PR TITLE
added auto-scroll function

### PR DESCRIPTION
**Pull request by vecchiotom**
added an auto scroll function with speed changeable by the user. the auto scroll will scroll from the top to the bottom of the page and then do the same from the bottom to the top, jquery was needed to make it work, so i also added some lines to implement it in every site the user will visit.

### CODE REVIEW

File: Scroll_Everywhere/scrolle.user.js [hunk 1]
```diff
@@ -134,3 +138,12 @@ function fFalse() {
 function slowF(x) {
     return 1 / (1 + Math.pow(Math.E, (-0.1 * x)));
 }
+
+function scroll(speed) {
```
Comment (vecchiotom) [inline Scroll_Everywhere/scrolle.user.js]: this is the main function, it has been written in jQuery, to minimize the lines of code to write

**Comment by tumpio** [comment]
Hi! This is interesting addition. There is one problem though. It overrides the original scroll function. It should be renamed for example to autoScroll.

How did you plan to call this function, a button or some mouse/key command?

**Comment by vecchiotom** [comment]
Hi! I didn't notice that the two functions had the same name XD sorry, and yes I planned to add a button in the in the lower left corner of every page but I'm still creating the style of the button so that it is pleasing to see.thanks you for commenting!

File: Scroll_Everywhere/scrolle.user.js [hunk 2]
```diff
@@ -10,6 +10,7 @@
 // @include         *
 // @grant           none
 // @version         0.3a
+// @require         https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js
```
Comment (vecchiotom) [inline Scroll_Everywhere/scrolle.user.js]: added this line instead of the javascript equivalent, to implement jQuery

Comment (vecchiotom) [inline Scroll_Everywhere/scrolle.user.js]: and also changed the name of the function to Autoscroll

**Pull request closed without merging**